webpack.config.js

```diff
@@ -10,7 +10,7 @@ Encore
     // directory where compiled assets will be stored
     .setOutputPath('public/build/')
     // public path used by the web server to access the output path
-    .setPublicPath('./build')
+    .setPublicPath('/build')
```

Comment:
@EstelleMyddleware , are you sure it is safe to push this change to main branch ?